cookbook/product_asset/sftp.rst

```diff
@@ -0,0 +1,118 @@
+
+This cookbook assumes that you already configured an external server. This cookbook should be made before the installation of the pim. Else you will have to migrate your data to your
+external server.
+
+Connect to an sftp server
+-------------------------
+
+Lets assume you already configured an sftp server. You will have to add a new adapter in the oneup_flysystem configuration file, with the sftp credentials.
+Once you created your new adapter you just have to set your new adapter in the filesystems category.
+
+.. code-block:: yaml
+
+    # /src/Acme/Bundle/EnrichBundle/Resources/config/bundles/oneup_flysystem.yml
+    oneup_flysystem:
+    adapters:
+        ...
+        oneup_flysystem:
+            adapters:
+                my_adapter:
+                    sftp:
+                        host: ftp.domain.com
+                        port: ~
+                        username: ~
+                        password: ~
+                        root: ~
+                        timeout: ~
+                        privateKey: ~
+                        permPrivate: ~
+                        permPublic: ~
+    filesystems:
+        asset_storage:
+            adapter: my_adapter
+            mount: assetStorage
+        catalog_storage:
+            adapter: my_adapter
+            mount: catalogStorage
+        tmp_storage:
+            adapter: tmp_storage_adapter
+            mount: tmpStorage
+            plugins:
+                - oneup_flysystem.plugin.list_files
+        archivist:
+            adapter: archivist_adapter
+            mount: archivist
+            plugins:
+                - oneup_flysystem.plugin.list_files
+
+
+For further information about sftp adapter check this link https://github.com/1up-lab/OneupFlysystemBundle/blob/master/Resources/doc/adapter_sftp.md
+
+Connect to an Amazon AwsS3v3 server
+-----------------------------------
+
+In order to use the AwsS3v3 adapter, you first need to create a client object defined as a service.
+This version requires you to use the "v4" of the signature.
+
+.. code-block:: yaml
+
+    # /src/Acme/Bundle/EnrichBundle/Resources/config/bundles/flysystem_clients.yml
+    services:
+    acme.s3_client:
+        class: Aws\S3\S3Client
+        factory_class: Aws\S3\S3Client
+        factory_method: factory
+        arguments:
+            -
+                version: '2006-03-01' # or 'latest'
+                region: "region-id" # 'eu-central-1' for example
+                credentials:
+                    key: "s3-key"
+                    secret: "s3-secret"
+
+Then Set this service as the value of the client key in the oneup_flysystem configuration.
+
+.. code-block:: yaml
+
+    oneup_flysystem:
+        adapters:
+            ...
+            acme.flysystem_adapter:
+                awss3v3:
+                    client: acme.s3_client
+                    bucket: ~
+                    prefix: ~
+    filesystems:
```

Comment:
it not good

the goal is to change ONLY the adapter
you don't have to touch the filesystems part as an integrator